lib/ui/platform_dispatcher.dart

```diff
@@ -308,6 +308,22 @@ class PlatformDispatcher {
     _invoke(onMetricsChanged, _onMetricsChangedZone);
   }
 
+
+  /// The framework will use this method to subscribe to platform focus changes.
+  ViewFocusChangeCallback? _onPlatformFocusChange;
+  set onPlatformFocusChange(ViewFocusChangeCallback callback) {
+    assert(_onPlatformFocusChange == null, 'onPlatformFocusChange was already binded');
+    _onPlatformFocusChange = callback;
```

Comment:
You'll likely also have to remember the zone in which the callback was set so that it can be invoked in that zone again. See how we do this for some of the other callbacks on PlatformDispatcher.

Reply:
Understood! Done.